godot/entity/player.gd

```diff
@@ -108,7 +108,7 @@ func _process(delta):
 	if direction.length() > 2:
 		direction = direction.normalized()
 	
-		velocity += direction * speed * delta
+		velocity += direction * speed * 1./60.
```

Comment:
The tutorial on using CharacterBody2D wasn't exactly clear on this. Apparently, you need to multiply with delta for non-linear movement, such as gravity, but not for linear movement like here.